strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/domain/GenericArtifactCoordinatesEntity.java

```diff
@@ -0,0 +1,91 @@
+package org.carlspring.strongbox.domain;
+
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import org.carlspring.strongbox.artifact.coordinates.GenericArtifactCoordinates;
+import org.carlspring.strongbox.data.domain.DomainEntity;
+import org.carlspring.strongbox.db.schema.Edges;
+import org.carlspring.strongbox.db.schema.Vertices;
+import org.neo4j.ogm.annotation.NodeEntity;
+import org.neo4j.ogm.annotation.Properties;
+import org.neo4j.ogm.annotation.Relationship;
+
+@NodeEntity(Vertices.GENERIC_ARTIFACT_COORDINATES)
+public class GenericArtifactCoordinatesEntity extends DomainEntity implements GenericArtifactCoordinates
+{
+    private String version;
+    @Properties
+    private final Map<String, String> coordinates = new LinkedHashMap<>();
+    @Relationship(type = Edges.EXTENDS, direction = Relationship.INCOMING)
+    private GenericArtifactCoordinates layoutArtifactCoordinates;
+
+    @Override
+    public void setHierarchyChild(GenericArtifactCoordinates child) {
+        this.layoutArtifactCoordinates = child;
+    }
```

Comment:
```suggestion
    
    private String version;
    
    @Properties
    private final Map<String, String> coordinates = new LinkedHashMap<>();
    
    @Relationship(type = Edges.EXTENDS, direction = Relationship.INCOMING)
    private GenericArtifactCoordinates layoutArtifactCoordinates;
    
    @Override
    public void setHierarchyChild(GenericArtifactCoordinates child)
    {
        this.layoutArtifactCoordinates = child;
    }
```

Reply:
@carlspring all of my code was formatted this way within past few years, if you want to change it then probably better will be to create a separate task because it will be a lot of changes.